components/Steps/index.jsx

```diff
@@ -230,15 +230,17 @@ export const inputLabels = {
       validation: {
         required: true,
         pattern: {
-          value: /^[0-9]{8}$/,
+          value: /^\d{8}$/,
         },
       },
     },
     businessIdentifyNumberVAT: {
+      type: 'number',
+      hint: 'Please enter your 9 digits number',
```

Comment:
We can follow the same pattern here: https://design.tax.service.gov.uk/hmrc-design-patterns/vat-registration-number/

It would be good to mention that the GB prefix is not needed.

```suggestion
      hint: 'Please enter your 9 digit number, without the \'GB\' at the start, for example 123456789. You can find it on your VAT registration certificate.',
```